### DISCUSSION TITLE
Environment preparation of Mac ｜ Mac系统环境配置

### DISCUSSION THREAD

**Discussion by zRzRzRzRzRzRzR** [THUDM/CogVLM#164] [Q&A]
在这里提出Mac环境下配置CogVLM模型的任何问题，例如：
+ Mac 电脑能不能运行这个模型
+ 我想知道CogVLM有没有针对M3芯片的优化推理方案

**Comment by thiner** [comment]
@zRzRzRzRzRzRzR 我正好想问你示例的两个问题。麻烦你解答一下。
我的Mac是M3 Max芯片，36GB统一内存。

**Reply by zsq2010** [reply]
mac m1 max,64G,
希望用到
